SPECS-EXTENDED/biosdevname/biosdevname.spec

```diff
@@ -162,10 +162,6 @@ make install install-data DESTDIR=%{buildroot}
 - Don't display _vf suffix on NPAR devices with single function
 - Fix PCIe/PIRQ slot mapping
 
-<<<<<<< HEAD
```

Comment:
Ohh How did we manage to sneak that in :(
Thanks for catching this

Reply:
No problem but I can't take credit for this find - Jon spot it. I'm just removing it.:)